### ISSUE TITLE
VSTS Account

### ISSUE THREAD

**Issue by DeluxZ** [gmasterscm/ama#29]
Where do you take credentials from to auto-login with VSTS? When I click the 'Login with VSTS' button I'm getting auto logged in with the wrong VSTS/Microsoft account, now I can't add our work VSTS as a remote. I have a 'Work account' and a 'personal account' with my work email address. We are using our 'personal account' for our work VSTS (VSTS isn't integrated with Active Directory yet). GMaster is using my 'Work account' which doesn't match with the account in VSTS.

When I try to add the VSTS remote server I get the following error: 'The requested account url name (https://<url>.visualstudio.com/) don't match with any logged account ()'

**Comment by psantosl** [comment]
Hi!
It is a known problem: we are using the default call to the API, but it seems if you are logged into windows with a Microsoft account, it is taking your Windows login instead.

Is this the case?

We'll try to make a change to solve this asap (which will be forcing to enter creds on sign up... which is painful but the only solution we know for cases like yours)

**Comment by DeluxZ** [comment]
Yes, that is the case. My windows account is linked to my Active Directory Work account which is different from my account I'm using for VSTS.

**Comment by DeluxZ** [comment]
@psantosl Any update on this ?
